scanner/mappers/mappers.go

```diff
@@ -302,9 +302,9 @@ func toProtoV4VulnerabilitiesMap(
 			repoID = v.Repo.ID
 		}
 		normalizedSeverity := toProtoV4VulnerabilitySeverity(ctx, v.NormalizedSeverity)
-		sev, err := severityAndScores(v, nvdScores)
+		sev, err := severityAndScores(v, nvdVulns)
 		if err != nil {
-			return nil, err
+			zlog.Warn(ctx).Str("vulnerability", v.ID).Msgf("failed to parse severity, ignoring: %v", err)
```

Comment:
should there be a `continue` here?

Reply:
No, because `severityAndScores()` might return partial data even when `err != nil`.

Reply:
I actually moved back the error and ignored empty severities in the mapping function `severityAndScore`. The reason is not to break the testing expectation that a bad payload will fail the whole vuln report call. I created a ticket to follow up on that: https://issues.redhat.com/browse/ROX-21842